ld/devices.data

```diff
@@ -497,7 +497,7 @@ lpc13 lpc13xx
 lpc13u lpc13xx USBRAM_OFF=0x20004000
 
 lpc17[56]x lpc17xx RAM1_OFF=0x2007C000 RAM2_OFF=0x20080000
```

Comment:
I presume this is wrong too then?

Reply:
No, this is correct. 
LPC17[78]x has a few things mapped to different addresses compared to all other LPC17xx:
- Peripheral SRAM
- GPIO
- AHB peripherals

Current libopencm3 will not work on LPC17[78]x, another sub-port or similar will need to be created. I'm currently trying to figure out how to implement this without duplicating everything.

Reply:
Probably needs to introduce a subdirectory like sam and stm32 I would suppose then.